docs/jmd/constructors.jmd

```diff
@@ -167,30 +222,30 @@ In this form optional arguments `fast` and/or `nAGQ` can be passed to the optimi
 
 As the name implies, `fast=true`, provides a faster but somewhat less accurate fit.
 These fits may suffice for model comparisons.
-```{julia;term=true}
-gm1a = fit!(GeneralizedLinearMixedModel(verbaggform, verbagg,
-    Bernoulli()), fast=true)
+```@example Main
+gm1a = fit(MixedModel, verbaggform, verbagg, Bernoulli(), fast = true)
 deviance(gm1a) - deviance(gm1)
-@time fit!(GeneralizedLinearMixedModel(verbaggform, verbagg,
-    Bernoulli()));
-@time fit!(GeneralizedLinearMixedModel(verbaggform, verbagg,
-    Bernoulli()), fast=true);
+@benchmark fit(MixedModel, $verbaggform, $verbagg, Bernoulli())
+@benchmark fit(MixedModel, $verbaggform, $verbagg, Bernoulli(), fast = true)
 ```
 
 The optional argument `nAGQ=k` causes evaluation of the deviance function to use a `k` point
 adaptive Gauss-Hermite quadrature rule.
 This method only applies to models with a single, simple, scalar random-effects term, such as
-```{julia;term=true}
-contra = MixedModels.dataset(:contra)
-contraform = @formula(use ~ 1+age+abs2(age)+livch+urban+(1|dist));
-@time gm2 = fit!(GeneralizedLinearMixedModel(contraform,
-    contra, Bernoulli()), nAGQ=9)
-@time deviance(fit!(GeneralizedLinearMixedModel(contraform,
-    contra, Bernoulli()), nAGQ=9, fast=true))
-@time deviance(fit!(GeneralizedLinearMixedModel(contraform,
-    contra, Bernoulli())))
-@time deviance(fit!(GeneralizedLinearMixedModel(contraform,
-    contra, Bernoulli()), fast=true))
+```@example Main
+contraception = MixedModels.dataset(:contra)
+contraform = @formula(use ~ 1 + age + abs2(age) + livch + urban + (1|dist));
+bernoulli = Bernoulli()
+deviances = Dict{Symbol,Float64}()
+b = @benchmarkable deviances[:default] = deviance(fit(MixedModel, $contraform, $contraception, $bernoulli));
+run(b)
+b = @benchmarkable deviances[:fast] = deviance(fit(MixedModel, $contraform, $contraception, $bernoulli, fast = true));
+run(b)
+b = @benchmarkable deviances[:nAGQ] = deviance(fit(MixedModel, $contraform, $contraception, $bernoulli, nAGQ=9));
+run(b)
+b = @benchmarkable deviances[:nAGQ_fast] = deviance(fit(MixedModel, $contraform, $contraception, $bernoulli, nAGQ=9, fast=true));
+run(b)
+sort(deviances)
```

Comment:
I don't think this sort is doing what you intended.  It seems to sort on the keys, not on the values.
```julia
julia> sort(deviances)
OrderedCollections.OrderedDict{Symbol,Float64} with 4 entries:
  :default   => 2372.73
  :fast      => 2372.78
  :nAGQ      => 2372.46
  :nAGQ_fast => 2372.51
```

Reply:
I intended on doing on keys as those as I ran them in that order but I probably want to sort them by performance now that I think about it.